app/views/sessions/unauthenticated.haml

```diff
@@ -1,5 +1,30 @@
-%h2 Unauthenticated
-%p Sorry! You don't appear to be a user in our system.
-%p If you are an ST dispatcher and believe you should
-be authorized, please contact your administrator.
-%br
+%h1
```

Comment:
Yeah, I did mean 'template'. I'd suggest looking through this with @Anbranin and figuring out what's actually needed here.